slack/backend/app/src/main/resources/config/application-postgres.yml

```diff
@@ -0,0 +1,9 @@
+spring:
+    datasource:
+        driver-class-name: org.postgresql.Driver
+        url: jdbc:postgresql://localhost:5444/gzac-core-db
+    jpa:
+        database-platform: org.hibernate.dialect.PostgreSQLDialect
+
+valtimo:
+    database: postgres
```

Comment:
I don't now when we switch over to postgres as the default. But remember that this than also needs to be updated